libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -36,21 +54,65 @@ export function toFragment(htmlStrings, ...values) {
   return fragment;
 }
 
-export const getFedsPlaceholderConfig = () => {
-  const { locale } = getConfig();
-  let libOrigin = 'https://milo.adobe.com';
+let federatedContentRoot;
+export const getFederatedContentRoot = () => {
+  if (federatedContentRoot) return federatedContentRoot;
+
   const { origin } = window.location;
+  const allowedOrigins = [
```

Comment:
I would move the values at the top of the file for easy update.
Also, is the any problems with allowing `*.adobe.com` ? I know we would need additional steps in Akamai, but might be easier to allow everything here.

Reply:
I'm not sure if we forgot any subdomain, so I'd like to default to www.adobe.com in those cases and take the small performance hit (extra TLS handshake), instead of potentially breaking any projects // end up without placeholders.